From 620d6704401d8c9538d590c7e8bfdd18cb33034c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20=C4=8Cern=C3=BD?= <jcerny@redhat.com>
Date: Wed, 14 Jun 2017 15:32:30 +0200
Subject: [PATCH] RHBZ#1461330: Add Anaconda remediation for rule
 "smartcard_auth"

Packages pam_pkcs11 and esc weren't installed by Anaconda during
installing, which caused that users can't log in.
---
 shared/templates/static/anaconda/smartcard_auth.anaconda | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 shared/templates/static/anaconda/smartcard_auth.anaconda

diff --git a/shared/templates/static/anaconda/smartcard_auth.anaconda b/shared/templates/static/anaconda/smartcard_auth.anaconda
new file mode 100644
index 000000000..fbe3aa984
--- /dev/null
+++ b/shared/templates/static/anaconda/smartcard_auth.anaconda
@@ -0,0 +1,3 @@
+# platform = multi_platform_rhel
+
+package --add=pam_pkcs11 --add=esc
